doc/gov/chapters/chapter01.rst

```diff
@@ -4,34 +4,45 @@ Introduction
+supporter, or observer. By participating, you automatically accept the individual anti-trust policies of LFN and GSMA,
+the joint Terms of Reference of LFN and GSMA the LFN Code of Conduct, as well as the LFN Trademark policy.
 
--  `Trademark Policy <https://lfprojects.org/policies/trademark-policy/>`__
+* `LFN Anti-Trust Policy <https://r.lfnetworking.org/Antitrust%20Slide.pdf>`__
+* `GSMA Anti-Trust Policy <https://github.com/cntt-n/CNTT/blob/master/Anti-Trust_Notice_GSMA_20190911.png>`__
+* `CNTT Terms of Reference (ToR) LFN \| GSMA <https://wiki.lfnetworking.org/display/LN/Terms+of+Reference?src=contextnavpagetreemode>`__
+   -  Note: If you are not self-employed, then you must ensure that your employer also agrees to the ToR before
+      participating.
+* `Code of Conduct LFN <https://lfprojects.org/policies/code-of-conduct/>`__
+* `Trademark Policy <https://lfprojects.org/policies/trademark-policy/>`__
 
 Recommended checklist for participating in the Anuket community:
 
--  Review all legal policies, terms, and conditions
--  Signup for an `LFN ID <https://sso.linuxfoundation.org/login/?state=g6Fo2SBEbDAzLVFFN18teXdLMWZlUlVmWlVrMFpwZElHaUFtTqN0aWTZIGxCNVo3NExGOVpxNzJGT25YRW5vSkxMTlAwTENaLVdSo2NpZNkgcFZ0QzZ2R2ZmMnFBd0lxOXZVWGlhdTZzN3h6OWR3UUI&client=pVtC6vGff2qAwIq9vUXiau6s7xz9dwQB&protocol=oauth2&audience=https:%2F%2Fapi-gw.platform.linuxfoundation.org%2F&scope=openid%20profile%20email&response_type=code&response_mode=query&nonce=VldNTTJZdzc1UX5HVzBFMFd0dXBzTlc1UXpyQnJpVFlzfnY5WHA3OTl6SQ%3D%3D&redirect_uri=https:%2F%2Fmyprofile.lfx.linuxfoundation.org&code_challenge=VzQLzce3RLtJjTo1QmtKvZ3hrYd6J5U1yyQ4FXVodZ4&code_challenge_method=S256&auth0Client=eyJuYW1lIjoiYXV0aDAtc3BhLWpzIiwidmVyc2lvbiI6IjEuMTMuMiJ9>`__
--  Signup for a `GitHub ID <https://github.com/signup?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F&source=header-home>`__
--  Signup for a `GitLab ID <https://gitlab.com/users/sign_up>`__
--  `Choose a sub-project <https://wiki.anuket.io/display/HOME/Projects>`__ and begin contributing; review the sub-project information and on how to participate including `meetings <https://wiki.anuket.io/display/HOME/Anuket+Meetings>`__.
--  Start engaging!
+*  Review all legal policies, terms, and conditions
+*  Signup for an `LFN ID <https://sso.linuxfoundation.org/login/?state=g6Fo2SBEbDAzLVFFN18teXdLMWZlUlVmWlVrMFpwZElHaUFtTqN0aWTZIGxCNVo3NExGOVpxNzJGT25YRW5vSkxMTlAwTENaLVdSo2NpZNkgcFZ0QzZ2R2ZmMnFBd0lxOXZVWGlhdTZzN3h6OWR3UUI&client=pVtC6vGff2qAwIq9vUXiau6s7xz9dwQB&protocol=oauth2&audience=https:%2F%2Fapi-gw.platform.linuxfoundation.org%2F&scope=openid%20profile%20email&response_type=code&response_mode=query&nonce=VldNTTJZdzc1UX5HVzBFMFd0dXBzTlc1UXpyQnJpVFlzfnY5WHA3OTl6SQ%3D%3D&redirect_uri=https:%2F%2Fmyprofile.lfx.linuxfoundation.org&code_challenge=VzQLzce3RLtJjTo1QmtKvZ3hrYd6J5U1yyQ4FXVodZ4&code_challenge_method=S256&auth0Client=eyJuYW1lIjoiYXV0aDAtc3BhLWpzIiwidmVyc2lvbiI6IjEuMTMuMiJ9>`__
```

Comment:
Jokingly, in line 28 we say "nothing to sign but CLA" and now we have a list of things to sign. Can we use "Create"?

Reply:
Create an IF ID? I would totally support that change.

Reply:
@pgoyal01 fixed in https://github.com/cntt-n/CNTT/pull/3129/commits/25fe075e515ab8191b149117ca863591ea84e48a